shared/constants/types/flux.js

```diff
@@ -10,6 +10,8 @@ export type TypedAction<T, P, E> = {
   payload: E,
 }
 
+export type NoErrorTypedAction<T, P> = TypedAction<T, P, P>
```

Comment:
I got tired of having to check for error conditions if we know we aren't going to ever set an error.